### PR TITLE
Feature/poap schedule

### PR DESCRIPTION
**Pull request by SlinkyPotato**
1. Add /poap schedule command

### CODE REVIEW

File: src/app/api/types/EventsRequestType.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { AxiosResponse } from 'axios';
```
Comment (nonsensecodes) [inline src/app/api/types/EventsRequestType.ts:1]: Can we put all the events types in one file?

File: src/app/utils/ServiceUtils.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import {
 	Collection,
 	Guild,
-	GuildMember,
+	GuildMember, Permissions,
```
Comment (nonsensecodes) [inline src/app/utils/ServiceUtils.ts:7]: Can we put `Permissions` on its own line for consistency?

File: src/app/api/types/EventsRequestType.ts (hunk 1 above)
Comment (amaredeus) [reply src/app/api/types/EventsRequestType.ts:1]: Would it be better if there was an events types folder?

Comment (nonsensecodes) [reply src/app/api/types/EventsRequestType.ts:1]: Perhaps! Basically a little better organization. The types themselves seemed small enough to warrant compiling into one file, but if you think a folder would be better with types modularized into their own separate files as a consistent practice, then we can do that 👍

**Review by nonsensecodes — APPROVED** [review]
I make one suggestion and get a total  code overhaul 😂 Great job, @amaredeus! Thank you for doing this 🙏

**Pull request merged**